doc/source/reference.rst

```diff
@@ -9,18 +9,19 @@ Drizzle (Image drizzling step)
     :maxdepth: 1
 
     reference/astrodrizzle
-    reference/adrizzle
-    reference/ablot
+    reference/updatenpol
```

Comment:
I am curious.  Since are reference links, I would have expected them to be in alphabetical order.  Once there are examples of use, the order of application would be clarified.  What is the rationale here?

Reply:
Yeah, I tried to organize things in the way that the steps are run. We can revert this to alphabetical order.

Reply:
I would suggest keeping these in alphabetical order since they are references.  Good Use Cases will allow one to see how to use the functions/methods in a useful way.

Reply:
I agree, I'll change these to have each section alphabetical.